packages/broadcast/component.js

```diff
@@ -1,46 +1,56 @@
-import { FabricWebComponent, windowExists } from '../utils';
+import { LitElement } from 'lit-element';
+import { windowExists } from '../utils';
+import { FabricToastContainer } from '../..';
 
-export class FabricBroadcast extends FabricWebComponent {
-  async connectedCallback() {
-    const REFETCH_INTERVAL = Number(this.getAttribute('interval')) || 300000; // 300 000 ms = 5 minutes
+export class FabricBroadcast extends LitElement {
+  static properties = {
+    _messages: {
+      state: true,
+      hasChanged(newVal, oldVal) {
```

Comment:
hasChanged ensures that the component render method will only be called if the data from the backend changes